android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactTVExoplayerView.java

```diff
@@ -943,6 +941,13 @@ public void onIsPlayingChanged(EventTime eventTime, boolean isPlaying) {
         if (isPlaying) {
             isPaused = false;
             startProgressHandler();
+        } else if (player != null) {
+            // When reach the endTime of limited seek range, we need to notify the JS side to show the restart layout.
+            // We will seek to start position at first, and then play() when user click the restart button.
+            if (LimitedSeekRange.isUseAsVod(player.getLimitedSeekRange()) && player.getPlaybackState() == Player.STATE_ENDED) {
+                player.seekTo(0);
```

Comment:
Do we need to seek to 0 right away? or only when the user has pressed restart?

Reply:
This is the result of my discussion with yaroslavlvov. 
For Vod program if we click the restart button JS side will reload this program via setSrc(), it is not appropriate for live channel. So we envision such a process (he will confirm with JS side):

When playing to endTime, the live channel will be paused. and then `seekto(0)`.
When the user clicks the restart button, we only needs JS side to call the existing `setPausedModifier(false)` interface. 

Of course, if we design a new interface, we can modify the current code so that seekto (0) occurs only when we click restart button.